apic_ml2/neutron/plugins/ml2/drivers/cisco/apic/network_constraints.py

```diff
@@ -64,13 +68,18 @@ def get_subnet_scope(self, tenant, network, cidr):
                         return constraints.get('default')
         return scope
 
+    def get_allow_vm_names(self, tenant, vrf):
+        if self.source:
+            return self.source.get_allow_vm_names(tenant, vrf)
+
 
 class ConfigFileSource(NetworkConstraintsSource):
     def __init__(self, config_file):
         self.config_file = config_file
         self.last_refresh_time = 0
         self.subnet_default_scope = None
         self.subnet_constraints = {}
+        self.allow_vm_names_list = {}
```

Comment:
nit: '_list' is superfluous and actually incorrect because this variable holds a dict. Can simply name this 'allowed_vm_names'